hypermap/search_api/views.py

```diff
@@ -6,6 +7,12 @@
 from .serializers import SearchSerializer
 import json
 
+import sys  
+
+#Set utf8 as coder and encoder string 
+reload(sys)  
+sys.setdefaultencoding('utf8')
```

Comment:
This is a no-no and we should avoid per http://stackoverflow.com/questions/3828723/why-should-we-not-use-sys-setdefaultencodingutf-8-in-a-py-script